docs/_static/header.rst

```diff
@@ -7,4 +7,6 @@
    pyfar <https://pyfar.readthedocs.io>
    sofar <https://sofar.readthedocs.io>
    spharpy <https://spharpy.readthedocs.io>
-   pyrato <https://pyrato.readthedocs.io>
\ No newline at end of file
+   pyrato <https://pyrato.readthedocs.io>
+   Help <https://pyfar-gallery.readthedocs.io/en/latest/help/index.html>
+   Contribute <https://pyfar-gallery.readthedocs.io/en/latest/community/index.html>
```

Comment:
```suggestion
   Help <https://pyfar-gallery.readthedocs.io/en/latest/help/help.html>
   Contribute <https://pyfar-gallery.readthedocs.io/en/latest/community/contributing.html>
```
If the structure wont change much, we could skip the likt to the index and directly go on the contributing and help page. This index inbetween would make scence if we have more than one subpage.

Reply:
I thought we might add a FAQ section to *Help* and the Code of Conduct to *Community*. Thats why I kept the TOC tree